maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/mkdir/SvnMkdirCommandTest.java

```diff
@@ -39,7 +42,7 @@
 
     String messageFileString;
 
-    protected void setUp()
+    public void setUp()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@Before` (in JUnit 4 style)?

Reply:
fixed